### ISSUE TITLE
[TF2] [Feature Request] Cow Mangler 5000 Idle Sound

### ISSUE THREAD

**Issue by BreavyTF2** [ValveSoftware/Source-1-Games#4644]
The cow mangler 5000 has an idle sound in the files that is unused.

**Comment by condor00fr** [comment]
If I remember coreectly the Richeous Bison also has an unused idle sound.

**Comment by pizzadox11** [comment]
guys you report idiotic stuff that arent even related to the subject, if you didnt notice.. this game is over 20GB, its PACKED,FULL of unused models/sounds and many more stuff, it fact it useful and it should not be removed

**Comment by condor00fr** [comment]
Exactly, a lot of the unused stuff is unused because Valve either forgots to add them or couldn't due to time contraints and/or not enough devs to finish thoses features. Like trowable bread for instance. They would have been available during game setups and whould have been able to be trowned at your mates. It's still in the game an is somewhat usable? Not too sure just how completed and how much of it remains.

Still, being able to re-Implement intended features that couldn't make it is a nice way to keep the game fresh either ways.
